query/stdlib/influxdata/influxdb/to.go

```diff
@@ -439,6 +440,9 @@ func (s Stats) Update(o Stats) {
 }
 
 func writeTable(t *ToTransformation, tbl flux.Table) error {
+	span, ctx := tracing.StartSpanFromContext(context.TODO())
```

Comment:
I'm not sure about this one.  Are you intending to eventually modify writeTable to accept a context?

Reply:
That would be the idea, yes. Again, query team's decision. When you touch I/O, it's a good place for tracing and a good place for context.